hashicorp/dev-portal
# Chore: Bump `mktg-logos` and `react-subnav` to latest

## 🔗 Relevant links



- [Preview link](https://dev-portal-git-BRANCH_NAME-hashicorp.vercel.app/PATH_TO_VIEW) 🔎
- 🎟️ [Asana Task](https://app.asana.com/0/1202791227659279/1203093711604797/f)

## 🗒️ What and 🤷 Why
It was recently discovered that the old Vault logo is currently being used in the `Subnav` component. Additionally, the `Subnav` component is currently being used on vaultproject.io. This PR updates packages so as to use the new logos (logo change [here](https://github.com/hashicorp/mktg-logos/pull/9)).

## 🛠️ How
- Updates to the latest versions of `react-subnav` and `mktg-logos`

## 🧪 Testing
- Navigate to [the preview](https://dev-portal-git-nels-choresbump-mktg-logos-and-subnav-hashicorp.vercel.app/)
- Select `Vault` from the products dropdown
- Verify that the correct logo is displayed in the `Subnav`:

_Incorrect Logo_
<img width="328" alt="Screen Shot 2022-10-04 at 2 34 41 PM" src="https://user-images.githubusercontent.com/39201593/193934645-e4e3df3d-6727-44a5-89f7-74aa351d800a.png">

_Correct Logo_
<img width="315" alt="Screen Shot 2022-10-04 at 2 34 35 PM" src="https://user-images.githubusercontent.com/39201593/193934635-69af28ee-3807-4514-ba5a-fcf6a954f8f8.png">


## 💭 Anything else?
